scripts/apollo_perf.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
+set -e
+
+TOP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${TOP_DIR}/scripts/apollo.bashrc"
+source "${TOP_DIR}/scripts/apollo_base.sh"
+
+function main() {
+    if ! command -v perf >/dev/null 2>&1; then
+      bash  "${TOP_DIR}/docker/scripts/install_perf.sh"
+      
+    fi
+    if [ -e /apollo-simulator ];then
+      cd /apollo-simulator
+      bash build.sh run >/dev/null 2>&1 & 
+      sleep 1
+      ps -ef | grep "replay_engine" | grep -v grep | awk '{print $2}' |  xargs sudo perf record  -g -p
```

Comment:
1. We can do this as a start. But better to find another clear way instead of `ps | grep` which may cause miss-hit silently.
2. Fix coding style. E.g.: There are several double-spaces. And the indents are mixed with 2-spaces and 4-spaces.

Reply:
the best way is: sudo perf record -g ./build.sh run. however this way run crash.